### PR TITLE
Add BiomeValues

### PR DESCRIPTION
**Pull request by aria1th**

### CODE REVIEW

File: src/main/java/me/senseiwells/essentialclient/clientscript/core/MinecraftAPI.java [hunk 1]
```diff
@@ -66,7 +67,8 @@ public class MinecraftAPI extends ValueTypes {
 		SCREEN = "Screen",
 		TEXT = "Text",
 		TRADE = "Trade",
-		WORLD = "World";
+		WORLD = "World",
+		BIOME = "Biome";
```
Comment (senseiwells) [inline src/main/java/me/senseiwells/essentialclient/clientscript/core/MinecraftAPI.java:71]: Alphabetical?

File: src/main/java/me/senseiwells/essentialclient/clientscript/values/BiomeValue.java [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+	}
+
+	@Override
+	public GenericValue<Biome> copy(Context context) {
+		return this;
+	}
+
+	@Override
+	public String getAsString(Context context) {
+		return "Biome{%s}".formatted(this.value.toString());
```
Comment (senseiwells) [inline src/main/java/me/senseiwells/essentialclient/clientscript/values/BiomeValue.java:35]: Should just use string concatenation here, `.formatted` is really slow, and `toString` is a very common operation

File: src/main/java/me/senseiwells/essentialclient/clientscript/values/BiomeValue.java [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+				MemberFunction.of("isCold", 1, this::isColdPos),
+				MemberFunction.of("isCold", 3, this::isColdFull),
+				MemberFunction.of("getId", this::getId),
+				MemberFunction.of("getSkyColor", this::getSkyColor),
+				MemberFunction.of("hasHighHumidity", this::hasHighHumidity)
+			);
+		}
+
+		@FunctionDoc(
+			name = "doesNotSnow",
```
Comment (senseiwells) [inline src/main/java/me/senseiwells/essentialclient/clientscript/values/BiomeValue.java:83]: `doesNotSnow` is a weird negative name, would have `canSnow` and invert it

File: src/main/java/me/senseiwells/essentialclient/clientscript/values/WorldValue.java [hunk 4]
```diff
@@ -1,639 +1,675 @@
-package me.senseiwells.essentialclient.clientscript.values;
```
Comment (senseiwells) [inline src/main/java/me/senseiwells/essentialclient/clientscript/values/WorldValue.java:1]: What happened here? Was this meant to be completely different, I can't see any changes so I'm not sure what happened here.

**Review by senseiwells — CHANGES_REQUESTED** [review]

File: src/main/java/me/senseiwells/essentialclient/clientscript/values/WorldValue.java (hunk 4 above)
Comment (aria1th) [reply src/main/java/me/senseiwells/essentialclient/clientscript/values/WorldValue.java:1]: Intellij error, or some weird thing.... not sure what happened

**Comment by senseiwells** [comment]
This looks good 👍. Ready for merge when I return.

**Review by senseiwells — APPROVED** [review]

**Comment by aria1th** [comment]
merged to #67

**Pull request closed without merging**